apps/xero/views.py

```
@@ -12,7 +12,9 @@
 from apps.xero.actions import get_xero_connector, sync_tenant
```

Ruff (0.8.2)

`apps.xero.actions.get_xero_connector` imported but unused

Remove unused import: `apps.xero.actions.get_xero_connector`

(F401)